eclipse/test/junit/saros/filesystem/EclipseReferencePointManagerTest.java

```diff
@@ -0,0 +1,215 @@
+package saros.filesystem;
+
+import org.easymock.EasyMock;
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.IPath;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import saros.activities.SPath;
+
+public class EclipseReferencePointManagerTest {
+
+  IProject project;
```

Comment:
Tests :+1: 
Is it really necessary to have all these fields accessible for each test or do some test just need a subset of the the result?

Please put only the general setup into a `@Before` annotated method and the test method specific setup into a separate private method which is called within the test or directly in the test method.

Reply:
I adjusted the whole tests. Every test creates  the needed mocks.